src/module/system/statistic/index.ts

```diff
@@ -271,24 +270,6 @@ class Statistic extends SimpleStatistic {
             breakdown,
             modifiers: modifiers.map((m) => m.toObject()),
         };
-
-        const rollable = options.rollable;
-        if (rollable) {
-            trace.roll = (args: RollParameters = {}): Promise<unknown> => {
-                const deprecationLabel = `Rolling ${this.label} (${this.slug}) via actor system data is deprecated`;
-                foundry.utils.logCompatibilityWarning(deprecationLabel, {
-                    since: rollable[0],
-                    until: rollable[1],
-                });
-
-                return this.check.roll({
-                    extraRollOptions: args.options ? [...args.options] : [],
-                    ...options,
-                });
-            };
-        }
-
-        return trace;
```

Comment:
Thoughts on keeping this around for when strikes get converted? This function was created as a way for us to convert things to statistic without breakage for a bit. We can always dig into git history to bring it back, so I'm not hardset either way.

Reply:
a shame you added a deprecation notice!